releaseNotes/Microsoft.Featuremanagement.AspNetCore.md

```diff
@@ -2,6 +2,10 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0-preview3 - April 10, 2024
+
+* Updated `Microsoft.FeatureManagement` reference to `4.0.0-preview3`. See the [release notes](./Microsoft.Featuremanagement.md) for more information on the changes.
```

Comment:
For all of these links, we could link to the release in github which shows the changes to all the packages. Might be easier to understand the changes when they're multi-package.